packages/notebook/src/widget.ts

```diff
@@ -1319,12 +1319,13 @@ export class Notebook extends StaticNotebook {
 
     this._activeCellIndex = newValue;
     const cell = this.widgets[newValue] ?? null;
-    if (cell !== this._activeCell) {
+    if (cell !== this._activeCell || newValue != oldValue) {
       // Post an update request.
       this.update();
       this._activeCell = cell;
       this._activeCellChanged.emit(cell);
     }
+
```

Comment:
Could you change this to avoid updating the UI if the active cell widget did not change?

```suggestion
    const cellChanged = cell !== this._activeCell;
    if (cellChanged) {
      // Post an update request.
      this.update();
      this._activeCell = cell;
    }
    
    if (cellChanged || newValue != oldValue) {
      this._activeCellChanged.emit(cell);
    }

```

Reply:
This screenshot looks fine to me. The problem previously was that both arrows were greyed out and on your screenshot only one is greyed out. Am I missing something?

Reply:
Yes you are right @krassowski . Sorry for the confusion.